creator-node/src/services/sync/syncImmediateQueue.js

```diff
@@ -84,12 +91,21 @@ class SyncImmediateQueue {
         )
       }
     )
+
     const prometheusRegistry = serviceRegistry?.prometheusRegistry
     if (prometheusRegistry !== null && prometheusRegistry !== undefined) {
       prometheusRegistry.startQueueMetrics(this.queue, worker)
     }
   }
 
+  async deleteOldActiveJobs() {
+    const oldActiveJobs = await this.queue.getJobs(['active'])
+    logger.info(
+      `[sync-immediate-processing-queue] removing ${oldActiveJobs.length} leftover active sync jobs`
+    )
+    await Promise.all(oldActiveJobs.map((job) => job.remove()))
```

Comment:
I'm hoping this re-uses the same redis connection and doesn't create, for example, millions of connections if there are millions of jobs to delete. I think we can merge but just calling out to move to chunked parallel if we run into issues

Reply:
but when you ran it locally, were there any active jobs?